pytorch_lightning/callbacks/stochastic_weight_avg.py

```diff
@@ -201,7 +201,7 @@ def on_train_epoch_start(self, trainer: "pl.Trainer", pl_module: "pl.LightningMo
             self.n_averaged = torch.tensor(0, dtype=torch.long, device=pl_module.device)
 
         if self.swa_start <= trainer.current_epoch <= self.swa_end:
-            self.update_parameters(self._average_model, pl_module, self.n_averaged, self.avg_fn)
+            self.update_parameters(self._average_model, pl_module, self.n_averaged, self._avg_fn)
```

Comment:
this is not tested anywhere? with this wrong ref it shall crash...

Reply:
Let's add a unit test to prevent regression. @qqueing Mind doing so ?